addons/BLUAlert/BLUAlert.lua

```diff
@@ -0,0 +1,89 @@
+--[[
+BLUAlert v1.20131102
+
+Copyright (c) 2015, Kainsin
```

Comment:
Please use `©` and the current year, unless you wrote this two years ago :)

You may have been using an old text as the template for the license. While the text didn't change, some formatting did. For the newest template, see [this page](https://github.com/Windower/Lua).